tarmac/src/main/scala/nl/lumc/sasc/biopet/pipelines/tarmac/Tarmac.scala

```diff
@@ -29,13 +37,76 @@ class Tarmac(val root: Configurable) extends QScript with PedigreeQscript with S
 
   class Sample(name: String) extends AbstractSample(name) {
 
-    val inputCountFile: Option[File] = config("count_file")
+    val inputXhmmCountFile: Option[File] = config("xhmm_count_file")
+    val inputWisecondorCountFile: Option[File] = config("wisecondor_count_file")
     val bamFile: Option[File] = config("bam")
 
+    /**
+     * Create XHMM count file or create link to input count file
+     * Precedence is given to existing count files.
+     * Returns a disjunction where right is the file, and left is
+     * a potential error message
+     */
+    lazy val outputXhmmCountFile: String \/ File = {
```

Comment:
Don't think using \/ here is really usefull, just use File and use Logging.addError inside here